control-plane/pkg/security/secret.go

```diff
@@ -50,6 +50,11 @@ const (
 	SaslPlain       = "PLAIN"
 	SaslScramSha256 = "SCRAM-SHA-256"
 	SaslScramSha512 = "SCRAM-SHA-512"
+
+	TlsEnabled  = "tls.enabled"
+	TlsCacert   = "ca.crt"
+	TlsUsercert = "user.crt"
+	TlsUserkey  = "user.key"
```

Comment:
```suggestion
        // Legacy Channel config to enable TLS, see https://github.com/knative-sandbox/eventing-kafka-broker/issues/2231
	SSLLegacyEnabled  = "tls.enabled"
```

We can remove some of them because:
- TlsCacert  == CaCertificateKey
- TlsUsercert == UserCertificate
- TlsUserkey == UserKey

Reply:
Done